Osmose/ribbit-recipes
# Add Mantine component library and Tabler icons.

- Mantine: https://mantine.dev/
- Tabler: https://tabler.io/icons

Replaces the placeholder app layout with Mantine's AppShell layout and moves the auth info into a header button + dropdown menu.

Note that signing out is now in a dropdown that appears when you click your username/avatar.

<img width="1009" alt="Screenshot 2024-04-12 at 7 49 21 PM" src="https://github.com/Osmose/ribbit-recipes/assets/193106/d6e703f6-5270-4c8f-a9b8-6bd19898a3c2">

## How to test
1. Pull in new deps with `npm install`
2. Run dev server with `npm run dev`
3. Load the app and check that signing in and out still functions.
